### ISSUE TITLE
current prereleased not installing in latest Azure VM (VS 2022 Enterprise in Windows Enterprise N)

### ISSUE THREAD

**Issue by anirugu** [microsoft/winget-cli#2435]
### Brief description of your issue

 I tried to create a VM in azure and remove all windows 11 apps and store was not there. I tried to install it using Add-AppPackage command in powershell it's not working

Add-AppPackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict validation.
Windows cannot install package Microsoft.DesktopAppInstaller_1.19.2161.0_x64__8wekyb3d8bbwe because this package
depends on a framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00.UWPDesktop" published
by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor
architecture and minimum version 14.0.30704.0, along with this package to install. The frameworks with name
"Microsoft.VCLibs.140.00.UWPDesktop" currently installed
Windows cannot install package Microsoft.DesktopAppInstaller_1.19.2161.0_x64__8wekyb3d8bbwe because this package
depends on a framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00.UWPDesktop" published
by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor
architecture and minimum version 14.0.30704.0, along with this package to install. The frameworks with name
"Microsoft.VCLibs.140.00.UWPDesktop" currently installed are:
{Microsoft.VCLibs.140.00.UWPDesktop_14.0.29231.0_x64__8wekyb3d8bbwe}
NOTE: For additional information, look for [ActivityId] a7621c07-d11f-0002-7119-706606acd801 in the Event Log or use
the command line Get-AppPackageLog -ActivityID a7621c07-d11f-0002-7119-706606acd801
At line:1 char:1
+ Add-AppPackage
+ ~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\anirud...bbwe.msixbundle:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

### Steps to reproduce

Use step mention in this video https://www.youtube.com/watch?v=J1hBZgRvZzI
then tried to install the msixbundle file (latest prereleased). it's failing with the given error.

### Expected behavior

It must install and work.

### Actual behavior

failing with the error.

### Environment

```shell
Tried in azure vm. you can do the same.
```

**Comment by denelon** [comment]
@anirugu we aren't fully supported on Windows Server yet (Windows Server 2022 should be considered experimental). We're building the business case with a community [discussion](https://github.com/microsoft/winget-cli/discussions/2361). There are some larger concerns around offline update scenarios. 

There has been a flurry of recent activity around predictable / reliable install behaviors. I'm looking to bring some of that work into our 1.4 release, as well as other work that needs to happen on Windows Server. Our goal is to make it possible to just install Windows and we're already there in the future, and well documented reliable mechanisms for automating the installation on other Windows SKUs.

**Comment by anirugu** [comment]
@denelon  I tried it on Windows 11 machine Enterprise N (VS 2022 Enterprise). It's not working there.

**Comment by denelon** [comment]
There are a couple of related Issues:
* https://github.com/microsoft/winget-cli/issues/2434
* https://github.com/microsoft/winget-cli/issues/2436

Is the "App Installer" preinstalled on that VM? Is the Microsoft Store enabled to get an update to "App Installer"?

**Comment by anirugu** [comment]
nope.  I run the script that remove all uwp bloatware. After  that I tried to run msixbundle file. As it's working for me on my machine, I start believing it will work on Windows PC (not server) VM too. Is Azure VM have anything different than the OS released in public.

**Comment by denelon** [comment]
It must. I'm not sure how those SKUs (Azure VMs) are configured.

**Comment by ItzLevvie** [comment]
> The frameworks with name "Microsoft.VCLibs.140.00.UWPDesktop" currently installed are:
> Microsoft.VCLibs.140.00.UWPDesktop_14.0.29231.0_x64__8wekyb3d8bbwe

You need the latest version of `Microsoft.VCLibs.140.00.UWPDesktop` from `https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx` = `Add-AppxPackage -Path https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx`

For `Microsoft.UI.Xaml.2.7` you can download from https://globalcdn.nuget.org/packages/microsoft.ui.xaml.2.7.2.nupkg and install `Microsoft.UI.Xaml.2.7.appx` from the `tools\AppX\x64\Release` folder.

Here's all required dependencies (x64): [WinGet dependencies.zip](https://github.com/microsoft/winget-cli/files/9304495/WinGet.dependencies.zip)

**Comment by denelon** [comment]
Bootstrapping WinGet when the store isn't present should be possible using the Repair-WinGetPackageManager cmdlet in the Microsoft.WinGet.Client PowerShell module.
